pkgs/by-name/tr/triton-llvm/package.nix

```diff
@@ -65,7 +65,7 @@ let
 in
 stdenv.mkDerivation (finalAttrs: {
   pname = "triton-llvm";
-  version = "21.0.0-git"; # See https://github.com/llvm/llvm-project/blob/main/cmake/Modules/LLVMVersion.cmake
+  version = "21.0.0-unstable-2025-06-10"; # See https://github.com/llvm/llvm-project/blob/main/cmake/Modules/LLVMVersion.cmake
```

Comment:
Nice catch, makes sense to me (I tunnel visioned too much on the llvm tag and forgot about the nixpkgs conventions).